packages/api-core/src/big-sky-plugin/fetchers.ts

```diff
@@ -0,0 +1,9 @@
+import { wpcom } from '../wpcom-fetcher';
+import type { BigSkyPluginResponse } from './types';
+
+export async function fetchBigSkyPlugin( siteId: number ): Promise< BigSkyPluginResponse > {
+	return await wpcom.req.get( {
+		path: `/sites/${ siteId }/big-sky-plugin`,
```

Comment:
Could we rename the entire directory as `site-big-sky-plugin` according to the path? See: 

https://github.com/Automattic/wp-calypso/blob/5a3cee0586de6576cadf38fa90395fe5ef68d548/packages/api-core/README.md?plain=1#L59-L60